server/src/consts.rs

```diff
@@ -0,0 +1,3 @@
+// constants
+
+pub const SECRET_SESSION_PREFIX: &str = "/org/freedesktop/secrets/session/";
```

Comment:
No need for this file at all. Just hardcode the value where it is used for now

Reply:
removed the file.